tests/ifs/unit/ifs_unittests.cpp

```diff
@@ -0,0 +1,386 @@
+/*
+ * Copyright 2023 Intel Corporation.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#if defined(__x86_64__) && defined(__linux__)
+
+#include "gtest/gtest.h"
+#include "sandstone.h"
+
+#undef log_warning
+#define log_warning
+#undef log_error
+#define log_error
+#undef log_info
+#define log_info
+#undef log_debug
+#define log_debug
+
+#include "../sandstone_ifs.c"
+
+#undef PATH_SYS_IFS_BASE
+#define PATH_SYS_IFS_BASE
+
+#include "../ifs.c"
+#include "ifs_test_cases.h"
+#include "ifs_unit_utils.h"
+
+struct cpu_info *cpu_info = nullptr;
```

Comment:
I'm defining those 2 here, but in the future they could interfere with other unittests that may defined them as well. Suggestions @lukasz-tuz @thiagomacieira ?

Reply:
I don't know. We can deal with that if we have a problem.